crates/experimentation_platform/src/api/experiments/handlers.rs

```diff
@@ -371,6 +399,16 @@ pub async fn conclude(
                 };
                 operations.push(ContextAction::MOVE((context_id, context_move_req)));
             } else {
+                let user_str = serde_json::to_string(&user).map_err(|err| {
```

Comment:
Do we want to send the whole user object in the headers or only the email address?

Reply:
Whole user, as what all data would be needed later on the context api or which api being called would not be known